initramfs-init.in

```diff
@@ -646,6 +652,22 @@ if [ -n "$ovlfiles" ]; then
 else
 	apk add --root $sysroot $repo_opt $apkflags $pkgs
 fi
+
+if [ -n "$ssh_install" ]; then
+	rc_add sshd default
+	echo "PermitRootLogin yes" >> $sysroot/etc/ssh/sshd_config
+	if [ -n "$ssh_key" ]; then
+		mkdir -p $sysroot/root/.ssh
+		wget "$ssh_key" -O $sysroot/root/.ssh/authorized_keys
+	elif [ -n "$ssh_pass" ]; then
+		echo "root:$ssh_pass" > $sysroot/.passwd
+		chroot $sysroot sh -c 'chpasswd < /.passwd'
+		rm $sysroot/.passwd
+	fi
+	# to make modloop happy
+	mkdir -p $sysroot/.modloop/modules/$(uname -r)
```

Comment:
Is this something that is only affecting openssh? I would believe it affects everything that uses `ALPINE_REPOS` and not only openssh.

Reply:
It's not critical though, just to make modloop init script not complainig during booting the installer.